src/libs/selectedAccount/selectedAccount.ts

```diff
@@ -238,6 +238,11 @@ export function calculateSelectedAccountPortfolio(
       const networkTotal = Number(result?.total?.usd) || 0
       newTotalBalance += networkTotal
 
+      if (network === 'gasTank' || network === 'rewards') {
+        const networkTokens = result?.tokens || []
+        tokens.push(...networkTokens)
+      }
```

Comment:
I think that the bug is larger and not limited to gasTank and rewards tokens. When doing this refactoring I haven't thought of the case where we have `latest` state but not `pending` state- in this case, there will be no tokens in `tokens`, as we are always looping through `pendingTokens`. I wrote the logic this way to limit the number of loops but it introduces this bug.

Reply:
Yep, this is after manually disabling pending updates
![image](https://github.com/user-attachments/assets/d28d4515-cc4f-4f2d-ba8c-3c695db78938)